### PR TITLE
[Merge with RIAI#372] Item fix circ restriction and authors limit

### PR DESCRIPTION
**Pull request by ntarocco**
* remove 4 weeks form item restrictions, already covered by the default
  no restriction value
* limit the number of authors for resolved documents

### CODE REVIEW

File: invenio_app_ils/documents/utils.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Document utils."""
+
+MAX_AUTHORS = 5
```
Comment (kpsherva) [inline invenio_app_ils/documents/utils.py:10]: nit: could it be overridable in the overlay?

File: invenio_app_ils/eitems/mappings/v6/eitems/eitem-v1.0.0.json [hunk 2]
```diff
@@ -33,69 +33,7 @@
...
-                },
-                "roles": {
-                  "type": "keyword"
-                },
-                "type": {
-                  "type": "keyword"
-                }
-              },
-              "type": "object"
+              "type": "text"
```
Comment (kpsherva) [inline invenio_app_ils/eitems/mappings/v6/eitems/eitem-v1.0.0.json:36]: I think it was a good decision not resolve the authors with the whole structure :)

**Review by kpsherva — APPROVED** [review]

File: invenio_app_ils/documents/utils.py (hunk 1 above)
Comment (ntarocco) [reply invenio_app_ils/documents/utils.py:10]: I thought to create a config for it, but then honestly I thought that it was just too much... do you prefer to have it a config?

Comment (kpsherva) [reply invenio_app_ils/documents/utils.py:10]: no I'm fine, it's not a blocking issue, and very low impact

**Pull request merged**